[CS2113T-T09-4] Simplst

LDerpy opened the pull request:
Simplst is a laptop app for managing warehouse inventory and processes, optimized for use via a Command Line Interface (CLI) for warehouse management workers.

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+#### Description
+The goods class keeps track of the various inventory that will be input into the system through the commands class.
+The diagram below shows the model component of the orderline class.
+![Good Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/Good.puml)
```
Consider adding a \<br> to move this diagram to a new line.
![image](https://user-images.githubusercontent.com/60597985/160961917-a481a26f-de1b-40e6-bc19-1bf4965a721b.png)

khseah replied:
I think the UML format does not really follow the standards we learnt in lecture. Eg we use "+", "-", "#" or "~" to indicate the visibilities, but in the UML diagram shown it used a red square and green circle

Also the return type should be at the end instead. Eg:
getId(): Int
instead of:
int getId()

khseah commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Good Class
+#### Description
+The goods class keeps track of the various inventory that will be input into the system through the commands class.
```
Can consider highlighting important words to aid readability
```suggestion
The `goods` class keeps track of the various inventory that will be input into the system through the commands class.
```

Lim-Shi-Han replied:
Yes as @khseah said, I also think denoting the class names as code would be good. In addition I think it would be better to capitalise it as well (e.g. `Good`)

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+![Regex Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/Regex.puml)
+The Class Diagram above shows the different variables and methods associated with the regex class.
+This class takes in the regular expression as a string and the string to be matched in the class contructor, and stores the results
+in the private variable `groupValues` which is a Hash Map where the key would be the capture group name and the value as the matched results.
```
Consider using `HashMap` in a code block instead of Hash Map to make it clear that it is a class.
![image](https://user-images.githubusercontent.com/60597985/160962185-3c4c1944-4848-4eae-80ef-e141e06b7f7e.png)

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+View Good belongs as part of the Commands Class. It is used when a user would like to know more information about an inventory item has an item id linked to it in the warehouse.
+
+#### Operation
+![View Good Sequence Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/viewGood.puml)
```
Consider making the return lines dotted lines.
![image](https://user-images.githubusercontent.com/60597985/160962380-3f78ccd1-5d84-48b7-9375-7ab627a5b87a.png)

khseah replied:
Return arrows should be dotted

eigne replied:
`User` is not a class, so the colon before `User` should not be included. Consider removing it.
![image](https://user-images.githubusercontent.com/60597985/160963256-3ede9091-d721-4a56-8371-884802eede86.png)

eigne replied:
The stickman figures are not covered in this module, so consider replacing them with boxes.
![image](https://user-images.githubusercontent.com/60597985/160964297-729be638-e8c1-4cfc-9946-a5a85eb03a97.png)

eigne replied:
The boxes + User at the bottom should not be present. Use an X instead to mark end of life for an object instead if necessary, or leave it empty.
![image](https://user-images.githubusercontent.com/60597985/160964468-4e594a1b-c645-490d-a2ca-89867c1cfb12.png)

Lim-Shi-Han replied:
Return lines should be dotted.

khseah commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+### Regex Class
+#### Description
+The Regex Class is created to simplify the use of regular expressions in Java.
+![Regex Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/Regex.puml)
```
This diagram indicates that a new Regex class is instantiated and the red marking shows that the activation bar is a constructor, which is not the case. You should add a dotted line before the activation bar
![Screenshot 2022-03-31 101327](https://user-images.githubusercontent.com/69412944/160962502-e0a549bb-5e2a-4538-9876-4e4cd0834e63.png)

Lim-Shi-Han replied:
I think there might be some issues with the `dummyClass` calling `Regex`, as it should be a new activation bar (and the previous activation bar should have ended prior). 

![image](https://user-images.githubusercontent.com/79437795/160971678-295bc037-1c3a-4e3d-bdb2-a0de3cad7e76.png)

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+
+1. The User input will be read by the User Interface Class
+2. The User Interface Class will the match the command keyword `list`
+3. Together with the previous step, the User Interface Class will check the flag followed by `list`
+   3.1 If the flag is `o/`, the User Interface Class will call the listOrders() method in the Warehouse class
+      3.1.1 Lastly, the list of orders is printed to the command line.
+   3.2 Else, if the flag is `g/`, the User Interface Class will call the listGoods() method in the Warehouse class
+      3.2.1 Following that, the Warehouse class will call the getGoods() method on the Good class
+      3.2.2 Lastly, the list of goods is printed to the command line.
```
Consider adding \<br> lines to properly format these numbered lists, as they currently appear as one big paragraph.
![image](https://user-images.githubusercontent.com/60597985/160962746-578f4c7f-86f0-47d6-95f6-5989f1b0cbd7.png)

darylhjd replied:
maybe some formatting error here!

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+4. This method will then iterate through orderLists and sum up each quantity of each Good object in each order.
+5. The totalGoods() method will return an integer of the number of goods in the warehouse. 
+
+For more examples of how a user can use a command, refer to the [UserGuide](/UserGuide.md)
```
This line seems unnecessary, but it could be more useful if it linked to this command's section specifically in the User Guide. Consider changing it or removing it.
![image](https://user-images.githubusercontent.com/60597985/160962960-d1b98d84-58a5-46b6-946c-e1798d882e20.png)

khseah commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
 
-## Design & implementation
-
+## Design
```
Can consider giving a high level overview of the architecture first, before diving deep into explaining each detail

khseah reviewed (commented):
Overall good implementation! One more thing to consider is to put the UML diagrams on a new line. Some UML diagrams are right after the sentence, causing formatting issues

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+
+### View Good Method
+#### Description
+View Good belongs as part of the Commands Class. It is used when a user would like to know more information about an inventory item has an item id linked to it in the warehouse.
```
It would be good to show the command format separately in text form as a code block, as well as an example. They are currently only visible in the sequence diagrams which makes it hard to copy and paste them, and is less obvious to the reader.

eigne commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+3. The Regex Class will then be called to match the rest of the user's input to find the values required to remove goods from the inventory.
+4. Afterwards, the User Interface class will call removeGood() method from the Commands Class
+5. This method will then reduce the quantity of a type of goods if the quantity input is not larger than the existing quantity. If the quantity input is the same as the existing quantity, the goods object will be removed from the inventory.
+6. The UI will show a message of format
```
Consider including more details on what methods are called and what types of objects are returned by each method, to make the implementation of this program clearer.

darylhjd commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+### Good Class
+#### Description
+The goods class keeps track of the various inventory that will be input into the system through the commands class.
+The diagram below shows the model component of the orderline class.
```
An `orderline` class is mentioned here, but there is no prior explanation or link to this class beforehand.

darylhjd commented on docs/DeveloperGuide.md:
````diff
@@ -4,26 +4,162 @@
+5. findMatch() will then return the HashMap object to groupValues variable
+
+#### Example of Usage
+```
+String regex = "id/(?<id>\\d*) n/(?<name>.*) q/(?<qty>\\d*) d/(?<desc>\\.*)";
+Regex regexMatch = new Regex(userInput, regex);
+HashMap<String, String> matches = regexMatch.getGroupValues();
+String id = matches.get("id");
+String name = matches.get("name");
+String quantity = matches.get("qty");
+String description = matches.get("desc");
+```
````
good that an example is given on how it works, but perhaps some explanation could be given for the regex pattern?

Lim-Shi-Han commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+4. Afterwards, the User Interface class will call removeGood() method from the Commands Class
+5. This method will then reduce the quantity of a type of goods if the quantity input is not larger than the existing quantity. If the quantity input is the same as the existing quantity, the goods object will be removed from the inventory.
+6. The UI will show a message of format
+    > `quantity` `name` have been removed
+to show that the operation is successful.
```
The formatting could be improved here

darylhjd commented on docs/DeveloperGuide.md:
````diff
@@ -4,26 +4,162 @@
+String description = matches.get("desc");
+```
+### Commands Class
+#### Description
+The Commands Class is the class which contains all the available commands that can be used in the Simplst CLI application.
+![Commands Class Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/CommandsClass.puml)
+
+More about how each feature is run can be seen in each of the methods that can be found below.
````
perhaps an explanation can be given for why this design of putting all commands into a single class was chosen rather than subclassing different commands.

Lim-Shi-Han commented on docs/DeveloperGuide.md:
```diff
@@ -4,26 +4,162 @@
+The list goods or list orders method belongs to the Warehouse Class. It is used to display the list of existing goods 
+or existing orders in the warehouse to the User.
+
+![List sequence Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/AY2122S2-CS2113T-T09-4/tp/master/docs/diagrams/List.puml)
+
```
The activation bar for the first condition should not spill over into the "else" condition

![image](https://user-images.githubusercontent.com/79437795/160970631-8623768c-b229-4a83-a229-f2d1628190a4.png)

Lim-Shi-Han reviewed (commented):
Looks good in general, slight changes to the formatting and diagrams would make it better!:)

Nnythingy committed:
Create OrderTest.java

Nnythingy committed:
Update OrderTest.java

Nnythingy committed:
Merge pull request #87 from Shyunyin/branch_ShyunYin-DG

Branch shyun yin dg

Nnythingy committed:
Merge branch 'master' into branch-Nnythingy-Version2.1

LDerpy committed:
Order class completion + other stuff

All features should be done except storage. Small fixes here and there included

Nnythingy committed:
Update Warehouse.java

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp

Nnythingy committed:
Update DG

Shyunyin committed:
Include exceptions for missing flags and empty non-optional fields

Shyunyin committed:
Edit Checkstyle

Shyunyin committed:
Edit checkstyle

Nnythingy committed:
Update UserGuide.md

Shyunyin committed:
Edit MatchKeywordsTest to fit new order input

Shyunyin committed:
Fix checkstyle

Shyunyin committed:
Edit MatchKeywordsTest regex for testing

Shyunyin committed:
Fix MatchKeywordTest

Shyunyin committed:
Fix MatchKeywordsTest again

Shyunyin committed:
FIx MatchKeywordstest

Shyunyin committed:
FIx MatchKeywordstest

Shyunyin committed:
Fix MatchKeywordsTest

Shyunyin committed:
Error for repeated SKU

tuan0369 committed:
Merge https://github.com/AY2122S2-CS2113T-T09-4/tp

Nnythingy committed:
Update Warehouse.java

Nnythingy committed:
Revert "Update Warehouse.java"

This reverts commit 29dce377ba5cd7546d05b76be1d9590847a0a1b1.

Nnythingy committed:
Update remove orderline feature

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Add diagrams for DG

Nnythingy committed:
Create CommandParser.puml

Nnythingy committed:
Delete CommandsClass.puml

Nnythingy committed:
Update DeveloperGuide.md

Nnythingy committed:
Update CommandParser.puml

Nnythingy committed:
Create Warehouse.puml

Nnythingy committed:
Update DeveloperGuide.md

hjunleon committed:
Adedd some serialize functions

LDerpy committed:
UG and DG updates

Merged so we can see on the github page how the guides look like and work from there. UG should be mostly done, DG is partly finished

LDerpy committed:
Set theme jekyll-theme-cayman

hjunleon committed:
Added serialize for two more cllasses

tuan0369 committed:
Add warehouse test

tuan0369 committed:
Update test for warehouse

tuan0369 committed:
Update warehouse test

tuan0369 committed:
Update warehouse test

tuan0369 committed:
Update warehouse test

Shyunyin committed:
Update src/main/java/seedu/simplst/Warehouse.java

Co-authored-by: Nnythingy <66124098+Nnythingy@users.noreply.github.com>

Nnythingy committed:
Update UserGuide

Nnythingy committed:
Update UserGuide.md

Commit:
Some UG and warehose changes

Shyunyin committed:
Include size in non-empty field

Shyunyin committed:
Finish UnitGoodTest setup OrderlineTest

LDerpy committed:
Set theme jekyll-theme-cayman

Nnythingy committed:
Merge branch 'branch-UGDG' into branch-Nnythingy-Version2.1

Nnythingy committed:
Merge pull request #92 from Shyunyin/branch-Shyunyin-skuoverwrite

Branch shyunyin skuoverwrite

tuan0369 committed:
Merge https://github.com/AY2122S2-CS2113T-T09-4/tp

tuan0369 committed:
Merge branch 'master' into branch-Tuan-UpdateDG

Nnythingy committed:
Update Order.java

Nnythingy committed:
Create Order.puml

Nnythingy committed:
Merge remote-tracking branch 'upstream/master' into branch-Nnythingy-Version2.1

LDerpy committed:
Merge pull request #3 from Nnythingy/branch-Nnythingy-Version2.1

Branch nnythingy version2.1

tuan0369 committed:
Add UserInterface class to DG

hjunleon committed:
Code almost complete

hjunleon committed:
Merged with master

Shyunyin committed:
Finish orderline test

tuan0369 committed:
Add Display class to DG

Nnythingy committed:
Update DeveloperGuide.md

Nnythingy committed:
Add protection to fulfilled orders

Nnythingy committed:
Update EXPECTED.TXT

Nnythingy committed:
Update EXPECTED.TXT

tuan0369 committed:
Update DG

Nnythingy committed:
Update DeveloperGuide.md

Nnythingy committed:
Merge branch 'branch-Nnythingy-Version2.1' of https://github.com/Nnythingy/tp into branch-Nnythingy-Version2.1

Nnythingy committed:
Merge pull request #93 from tuan0369/branch-Tuan-Warehouse-Test

Branch tuan warehouse test

Shyunyin committed:
Include parser tests

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-RegexMatching

# Conflicts:
#	src/main/java/seedu/simplst/parsers/AddParser.java

Shyunyin committed:
Fix conflicts

Shyunyin committed:
Fix checkstyle

Shyunyin committed:
Fix checkstyle

hjunleon committed:
Save and restore works

Nnythingy committed:
Update DeveloperGuide.md

hjunleon committed:
Merge branch 'master' into branch-JunLeong-storage

hjunleon committed:
Merge pull request #97 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

Shyunyin committed:
Commented out matchKeywordsTest

Nnythingy committed:
Update DeveloperGuide.md

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-RegexMatching

# Conflicts:
#	src/main/java/seedu/simplst/Display.java

Shyunyin committed:
Fix UnitGoodTest

Commit:
UG updates and some display to UI changes

hjunleon committed:
Formatted code

hjunleon committed:
Merge pull request #99 from hjunleon/branch-JunLeong-storage

Formatted code

Commit:
More updates

LDerpy committed:
Merge branch 'master' into branch-Shyunyin-RegexMatching

Commit:
Text ui fix

LDerpy committed:
Remove duplicate method

There was another tryCommandAgain. Sorry my mistake when resolving just now.

Commit:
More fixes

LDerpy committed:
Shyun match keywords fix

LDerpy committed:
Tuan DG updates

Nnythingy committed:
Merge branch 'master' into branch-Shyunyin-UnitTest

Shyunyin committed:
Update addParserTest

LDerpy committed:
Shyun tests

LDerpy committed:
Merge branch 'master' into branch-UGDG

LDerpy committed:
UG and some other minor changes

hjunleon committed:
Strengthened the storage code

Nnythingy committed:
Merge branch 'master' into branch-Nnythingy-Version2.1

Nnythingy committed:
Update PPP

Nnythingy committed:
Merge branch 'branch-Nnythingy-Version2.1' of https://github.com/Nnythingy/tp into branch-Nnythingy-Version2.1

Nnythingy committed:
Update AboutUs.md

Nnythingy committed:
Update DeveloperGuide.md

Nnythingy committed:
Update Display.puml

Nnythingy committed:
Update UserInterface.puml

Nnythingy committed:
Update ViewOrder.puml

Nnythingy committed:
Fix Checkstyles

Nnythingy committed:
Update Warehouse.java

Nnythingy committed:
Update Display.java

Nnythingy committed:
Update AddParser.java

hjunleon committed:
Strengthened the storage code

hjunleon committed:
Merge branch 'master' into branch-JunLeong-storage

hjunleon committed:
Merge pull request #100 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

Shyunyin committed:
AddParserTest for UnitGood, Good and Order

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-UnitTest

# Conflicts:
#	src/main/java/seedu/simplst/Warehouse.java
#	src/main/java/seedu/simplst/parsers/AddParser.java
#	src/main/java/seedu/simplst/parsers/CommandParser.java
#	src/main/java/seedu/simplst/parsers/FindParser.java
#	src/main/java/seedu/simplst/parsers/FulfillParser.java
#	src/main/java/seedu/simplst/parsers/HelpParser.java
#	src/main/java/seedu/simplst/parsers/ListParser.java
#	src/main/java/seedu/simplst/parsers/RemoveParser.java
#	src/main/java/seedu/simplst/parsers/TotalParser.java
#	src/main/java/seedu/simplst/parsers/ViewParser.java

Shyunyin committed:
Update FindParserTest

Shyunyin committed:
Update HelpParserTest

hjunleon committed:
Fixed save file error handling

Shyunyin committed:
Update list parser

Shyunyin committed:
Update RemoveParserTest

Commit:
Update guides and some small fixes

Commit:
Some checkstyle fixes

LDerpy committed:
Merge pull request #102 from LDerpy/branch-2.1SmallFixes

Update guides and some small fixes

LDerpy committed:
Merge branch 'master' into branch-Nnythingy-Version2.1

Shyunyin committed:
Update TotalParserTest

LDerpy committed:
Merge pull request #96 from Nnythingy/branch-Nnythingy-Version2.1

Branch nnythingy version2.1

Shyunyin committed:
Update ViewParserTest

Shyunyin committed:
Fix WarehouseTest

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-UnitTest

# Conflicts:
#	src/main/java/seedu/simplst/parsers/AddParser.java

Shyunyin committed:
Fix checkstyle

Commit:
Added Exiting and a small FAQ for UG alongside PPP and DG formatting fixes

LDerpy committed:
Merge pull request #103 from LDerpy/branch-UGDGPPP

Documentation stuff

Shyunyin committed:
Include quickstart picture in UG

Shyunyin committed:
Include mainstream OS to glossary of DG

Shyunyin committed:
Include symbol for order commands for managers in UG

hjunleon committed:
Added tests and polished abit of code

hjunleon committed:
merge conflicts

Shyunyin committed:
Fix checkstyle

Nnythingy committed:
Create FulfillSequence.puml

Nnythingy committed:
Update ViewOrder.puml

Nnythingy committed:
Update DeveloperGuide.md

Nnythingy committed:
Update DeveloperGuide.md

tuan0369 committed:
Update code

tuan0369 committed:
Update display

LDerpy committed:
Merge pull request #101 from Shyunyin/branch-Shyunyin-UnitTest

Branch shyunyin unit test

LDerpy committed:
Merge pull request #104 from Shyunyin/branch-Shyunyin-OtherDocumentation

Include quickstart picture in UG

LDerpy committed:
Merge branch 'master' into branch-JunLeong-storage

tuan0369 committed:
Add PPP

hjunleon committed:
Adds the isCheckedOff and qtyFulfillled

hjunleon committed:
Merged

hjunleon committed:
Merge branch 'branch-JunLeong-storage' of github.com:hjunleon/tp into branch-JunLeong-storage

hjunleon committed:
Update UserGuide.md

tuan0369 committed:
Update PPP

tuan0369 committed:
Merge https://github.com/AY2122S2-CS2113T-T09-4/tp

tuan0369 committed:
Merge branch 'master' into branch-Tuan-UpdatePPP

Nnythingy committed:
Update AboutUs.md

LDerpy committed:
Update AboutUs.md

LDerpy committed:
Merge pull request #106 from Nnythingy/branch-Nnythingy-Version2.1

DG and Portfolio updates

Shyunyin committed:
Include acknowledgement in DG

Nnythingy committed:
Merge branch 'master' into branch-Nnythingy-Version2.1

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-OtherDocumentation

Shyunyin committed:
Change format of acknowledgement

Shyunyin committed:
Edit acknowledgement

Nnythingy committed:
Update nnythingy.md

LDerpy committed:
Merge pull request #105 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

LDerpy committed:
Merge pull request #109 from Nnythingy/branch-Nnythingy-Version2.1

Branch nnythingy version2.1

tuan0369 committed:
Fix checkstyle

Nnythingy committed:
Update KeywordMatching.puml

tuan0369 committed:
Merge branch 'master' into branch-Tuan-UpdatePPP

hjunleon committed:
DG stuff

hjunleon committed:
Merge branch 'branch-JunLeong-storage' of github.com:hjunleon/tp into branch-JunLeong-storage

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-OtherDocumentation

Nnythingy committed:
Update DeveloperGuide.md

tuan0369 committed:
Fix checkstyle

tuan0369 committed:
Merge branch 'branch-Tuan-UpdatePPP' of https://github.com/tuan0369/tp into branch-Tuan-UpdatePPP

Nnythingy committed:
Update Simplst.java

Nnythingy committed:
Update Warehouse.java

Nnythingy committed:
Update EXPECTED.TXT

tuan0369 committed:
Update PPP

tuan0369 committed:
Merge pull request #108 from tuan0369/branch-Tuan-UpdatePPP

Add PPP

hjunleon committed:
Added diagrams

Commit:
Checkstyle and bug fixes

LDerpy committed:
Merge branch 'master' into branch-LastMinuteStuffv2.1

hjunleon committed:
Some DG stuff

Shyunyin committed:
Include PPP for Shyunyin

hjunleon committed:
Dealt with merge issues

Commit:
Checkstyle fix

Nnythingy committed:
Merge pull request #107 from Shyunyin/branch-Shyunyin-OtherDocumentation

Include acknowledgement in DG

Nnythingy committed:
Merge pull request #111 from Nnythingy/branch-Nnythingy-Version2.1

Branch nnythingy version2.1

Nnythingy committed:
Merge pull request #112 from Shyunyin/branch-Shyunyin-PPP

Include PPP for Shyunyin

Commit:
Checkstyle fixes

Commit:
checkstyle fix final v2.1

hjunleon committed:
Merge pull request #113 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

Commit:
More checkstyle fixes

Nnythingy committed:
Merge branch 'master' into branch-LastMinuteStuffv2.1

hjunleon committed:
double checking

Commit:
Test fix

Commit:
Merge branch 'branch-LastMinuteStuffv2.1' of https://github.com/LDerpy/tp into branch-LastMinuteStuffv2.1

Commit:
UG stuff

Shyunyin committed:
Update links in about us

Shyunyin committed:
Merge pull request #114 from Shyunyin/branch-Shyunyin-AboutUs

Update links in about us

Nnythingy committed:
Update RemoveParser.java

hjunleon committed:
Added PPP and DG updates

Nnythingy committed:
Merge pull request #115 from Nnythingy/master

Update RemoveParser.java

hjunleon committed:
Merged

hjunleon committed:
Merge pull request #116 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

Commit:
More fixes

LDerpy committed:
Merge branch 'master' into branch-LastMinuteStuffv2.1

Commit:
test test

Nnythingy committed:
Update Warehouse.java

Commit:
Merge branch 'branch-LastMinuteStuffv2.1' of https://github.com/LDerpy/tp into branch-LastMinuteStuffv2.1

Commit:
final final fix

Nnythingy committed:
Merge pull request #117 from Nnythingy/master

Update Warehouse.java

Commit:
true final fix

Commit:
text ui stuff final?

hjunleon committed:
Added one orderSerialize sequence, will add just one restore for example

hjunleon committed:
Merge branch 'master' into branch-JunLeong-storage

Commit:
pls fix

Commit:
UG update

LDerpy committed:
Merge pull request #110 from LDerpy/branch-LastMinuteStuffv2.1

Checkstyle and bug fixes

hjunleon committed:
Final DG Push

hjunleon committed:
Added last diagram

Commit:
UG update

hjunleon committed:
Merge branch 'master' into branch-JunLeong-storage

LDerpy committed:
Merge pull request #119 from LDerpy/branch-UGLAST

UG update

LDerpy committed:
Merge pull request #118 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

hjunleon committed:
Typo

hjunleon committed:
Merge branch 'master' into branch-JunLeong-storage

Nnythingy committed:
Merge pull request #120 from hjunleon/branch-JunLeong-storage

Branch jun leong storage

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Update UserGuide.md

Shyunyin committed:
Merge pull request #121 from Nnythingy/master

2101 userguide updates

Shyunyin committed:
Include class definition

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp

# Conflicts:
#	src/main/java/seedu/simplst/Simplst.java

Shyunyin committed:
Edit some parts of UG

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Update UserGuide.md

Shyunyin committed:
Include FAQ

Shyunyin committed:
Merge pull request #125 from Nnythingy/master

adding tips for UG

Shyunyin committed:
Merge branch 'master' of https://github.com/AY2122S2-CS2113T-T09-4/tp into branch-Shyunyin-2101UG1

# Conflicts:
#	docs/UserGuide.md

Nnythingy committed:
Merge pull request #124 from Shyunyin/branch-Shyunyin-2101UG1

Branch shyunyin 2101 ug1

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Merge remote-tracking branch 'upstream/master'

Nnythingy committed:
Update UserGuide.md

Nnythingy committed:
Update StorageTest.java

Nnythingy committed:
Merge pull request #126 from Nnythingy/master

Update UserGuide.md

Nnythingy committed:
Revert "Update StorageTest.java"

This reverts commit e719c1dfd029b994061d47fee35f933b0fe84be7.

Nnythingy committed:
Merge pull request #127 from Nnythingy/master

Revert "Update StorageTest.java"
